tests/data_files/print_c.pl

```diff
@@ -1,34 +0,0 @@
-#!/usr/bin/env perl
-use strict;
-use warnings;
-
-if (!@ARGV || $ARGV[0] == '--help') {
-    print <<EOF;
-Usage: $0 mbedtls_test_foo <file.pem
-       $0 TEST_FOO mbedtls_test_foo <file.pem
-Print out a PEM file as C code defining a string constant.
-
-Used to include some of the test data in /library/certs.c for
```

Comment:
I don't understand what you mean about `format_rsa.key` and `rsa_pkcs8_1024_public.der`. Is `format_rsa.key` the private key for `rsa_pkcs8_1024_public.der`? They weren't added together, but still may be related.

- `format_rsa.key` was added with ed56b224dea9ba2513b8199cc405705340af9b49 ("- Added support for PKCS#8 wrapper on reading private keys (Fixes ticket #20)".
- `rsa_pkcs8_1024_public.der` was added with d0c56de93415a44671716dc927e1033d038ab913 ("Add support for public keys encoded with PKCS#1") (git thinks as `format_gen_der.pub` later renamed to ``rsa_pkcs8_1024_public.der` via b00651828943a505315e5ca1de0cdd6c96e449ce "Resolve PR review comments\n...rename the key files, according to their type and key size", from merge e6844ccf2b83)

Is there any value in keeping `format_rsa.key`? Perhaps yes, as private key for public keys we still use in testing.

Is there any value in keeping `rsa_pkcs8_1024_public.der`? No tests use this.

Reply:
Yes, `format_rsa.key` is the private key for `rsa_pkcs8_1024_public.der`. We shouldn't remove the private key if we keep the public key. `format_gen.key` and `format_pkcs12.fmt` are the same key in a different format.

Reply:
This PR currently removes `rsa_pkcs8_1024_public.der`, since no tests use it. I'm not sure how to see which private keys are for each public key we are keeping. If I've messed up, we can get the private key from earlier in git history.